src/wiki/ventoy.md

```diff
@@ -0,0 +1,106 @@
+:::
+
+## Downloading windows
+### Downloading the tool for downloading the ISO
+
+1. For Windows 10, go to [https://www.microsoft.com/software-download/windows10](https://www.microsoft.com/software-download/windows10) <br>
+   For Windows 11, go to [https://www.microsoft.com/software-download/windows11](https://www.microsoft.com/software-download/windows11)
+
+2. Click the **Download Tool Now** button.
+
+   ![](./img/downloading-windows/downloadtoolnow.png)
+2. Wait for the installer to prepare the files needed.
+3. After the tool has been downloaded, run it.
+
+### Using the tool
+
+1. The first thing you will see is this screen, click Accept.
+Select **Create installation media for another PC**, then press next.
+   ![](./img/downloading-windows/licenseterms.png)
+
+2. The next screen you see will be this, wait for the installer to complete preparation.
+4. Use the recommended settings, or select your preferred Language, Edition, and Architecture.
+   ![](./img/downloading-windows/gettingthings.png)
+::: tip Note
+If you're planning to install Windows on another PC and don't know what architecture it uses, then select "Both".
+:::
+   — You need to click on:**want to reinstall Windows on another computer** click **Create installation media**.
+
+    ![](./img/downloading-windows/whatdoyouwanttodo.png)
+5. Select **ISO file**, then press next.
+4. Select the correct Architecture, Edition, and Language of Windows, or use the recommended settings.
+
+   ::: tip Note
+   If you're planning on installing Windows on another PC and don't know what to select for the architecture, then choose both x86 and x64.
+   :::
+
+   ![](./img/downloading-windows/changelanguagearchitecture.png)
+Once the file has been downloaded, copy it to the Ventoy Drive.
+5. On the Media Creation Tool, select **ISO file**.
+
+You can read more about reinstalling Windows [here](installing-windows).
+
+6. Select a path to save to.
+## Downloading Antiviruses
+If you want to detect and remove any malware or rootkits that your computer has been infected with, you can try the following options.
+
+ - Microsoft Defender Offline: [32-bit](https://go.microsoft.com/fwlink/?LinkID=234123), [64-bit](https://go.microsoft.com/fwlink/?LinkID=234124)
+
+Download the ISO, and copy it to your Ventoy Drive.
+The instructions for installing or reinstalling Windows in your pc are here: [Installing Windows](installing-windows).
+:::
+
+## Download the AV
+If you want to detect rootkits and malware that got infected in your computer, and you want to clean them, use you can use one you've here listed:
+ - Kaspersky: https://www.kaspersky.com/downloads/free-rescue-disk (Not usable in US).
+ - Microsoft Defender Offline: https://support.microsoft.com/en-us/windows/help-protect-my-pc-with-microsoft-defender-offline-9306d528-64bf-4668-5b80-ff533f183d6c (head into Windows 8.1 and 7 to know how to download your .iso).
```

Comment:
We just download it earlier though. Is this a documentation link? Perhaps add it to the section above for MDO.
```suggestion
```